RetrospectiveExtension.Frontend/config/environment.tsx

```diff
@@ -1,5 +1,5 @@
 const environment = {
-  CollaborationStateServiceUrl: '<collaboration state service url>',
+  CollaborationStateServiceUrl: 'https://app-nbrtx.azurewebsites.net/',
```

Comment:
my current PR will add `enviornment.tsx` to the `.gitignore` list and create a templat file. This waywe do not accidentally upload secrets or other config values